src/node/index.ts

```diff
@@ -198,8 +200,27 @@ class Node {
           this.logger.info('Point node process exited')
         })
         .catch(e => {
+          // We only give special handling to `point error codes`, which are > 1.
+          // TODO: for now, we hardcode the codes we want to handle,
+          // we'll improve this when we have the `point-error-codes` shared repo.
+          if ([11, 12, 13].includes(e.code)) {
+            // Critical error from Point Engine, stop the ping interval as they are unrecoverable.
```

Comment:
I think, with our new retry logic, we shouldn't clear it here. We handle clearing it another way.